src/runtime/slot-dom-consumer.ts

```diff
@@ -26,13 +27,9 @@ export class SlotDomConsumer extends SlotConsumer {
     this._observer = this._initMutationObserver();
   }
 
-  constructRenderRequest(hostedSlotConsumer): string[] {
+  constructRenderRequest(): string[] {
```

Comment:
I'm trying to remember why we originally wanted to prefix the hosted-slot templates with the transformation particle prefix, but i think we can no longer do that, is it not needed anymore?
sorry, this is a very vague question, i'll dig it myself later. also there is an issue i file https://github.com/PolymerLabs/arcs/issues/2572 to reconsider the whole way we fetch and cache templates. let's discuss separately.

Reply:
It looks like we never passed anything in the constructRenderRequest(), so this is at least this cleanup is not making anything worse...